lib/ChainHandler.js

```diff
@@ -0,0 +1,46 @@
+/* @flow weak */
+"use strict";
+
+var async = require("async");
+
+var ChainHandler = module.exports = function ChainHandler() { };
+
+ChainHandler.prototype.chain = function(otherHandler) {
```

Comment:
I'm wondering if calling the parameter `nextHandler` would make clearer the the order of invocation... Just a suggestion though, it's fine as is.